lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/columns.scss

```diff
@@ -0,0 +1,219 @@
+// Inspired by https://bulma.io/documentation/columns/
+
+.columns {
+  @media screen and (min-width: $breakpoint-sm) {
+    display: flex;
+  }
+
+  margin-left: calc(-1 * var(--space-sm));
+  margin-right: calc(-1 * var(--space-sm));
+  margin-top: calc(-1 * var(--space-sm));
+
+  &:not(:last-child) {
+    margin-bottom: calc(var(--space-lg) - var(--space-sm));
+  }
+
+  // Sizing Variables
+  --four-fifths: calc(100% * 4/5);
+  --three-fifths: calc(100% * 3/5);
+  --two-fifths: calc(100% * 2/5);
+  --one-fifth: calc(100% * 1/5);
+
+  --three-quarters: calc(100% * 3/4);
+  --one-quarter: calc(100% * 1/4);
+
+  --two-thirds: calc(100% * 2/3);
+  --one-third: calc(100% * 1/3);
+
+  --one-half: calc(100% * 1/2);
+
+  // 12 Space Sizings
+  --one-twelfth: calc(100% / 12 * 1);
+  --two-twelfths: calc(100% / 12 * 2);
+  --three-twelfths: calc(100% / 12 * 3);
+  --four-twelfths: calc(100% / 12 * 4);
+  --five-twelfths: calc(100% / 12 * 5);
+  --six-twelfths: calc(100% / 12 * 6);
+  --seven-twelfths: calc(100% / 12 * 7);
+  --eight-twelfths: calc(100% / 12 * 8);
+  --nine-twelfths: calc(100% / 12 * 9);
+  --ten-twelfths: calc(100% / 12 * 10);
+  --eleven-twelfths: calc(100% / 12 * 11);
+  --twelve-twelfths: 100%;
+}
+
+// Wrapping
+.columns.is-multiline {
+  flex-wrap: wrap;
+}
+
+// Gapless
+.columns.is-gapless {
+  margin-left: 0;
+  margin-right: 0;
+  margin-top: 0;
+
+  &>.column {
+    margin: 0;
+    padding: 0!important;
+  }
+}
+
+// Variable Gap Sizing
+.columns.is-variable {
+  --columnGap: var(--space-sm);
+  margin-left: calc(-1 * var(--columnGap));
+  margin-right: calc(-1 * var(--columnGap));
+
+  // TODO: better name? These could be confused with the column sizings
+  // E.G. `.columns.is-variable.is-1` is close to `.column.is-1`
```

Comment:
Any ideas on a better pattern for this?

Reply:
`.is-gap-one`?

Reply:
Yeah, I think being more explicit with the class would make it easier to read and pick up for new devs. This would also match with current utility class methods

```css
.is-gap-none
.is-gap-xxs
.is-gap-xs
.is-gap-sm
etc...
```